Makefile

```diff
@@ -10,6 +10,8 @@ SHELL   := /bin/bash
 ## notice and this notice are preserved.  This file is offered as-is,
 ## without any warranty.
 
+TAR := $(shell which gtar 2>/dev/null || echo "tar")
```

Comment:
Quotes aren't really needed, `echo tar` should be fine

Reply:
Oops, yep. That's just a habit of mine: I like them to show syntax-highlighted as strings in my text editor.

Reply:
Removed quotes.